src/services/sendEmail.service.js

```diff
@@ -1,9 +1,8 @@
-
 /* eslint-disable require-jsdoc */
 import sgMail from '@sendgrid/mail';
 import dotenv from 'dotenv';
 import mailGen from 'mailgen';
-import { verifyMessage } from '../utils/emailMessages';
+import { verifyMessage, userRoleMessage } from '../utils/emailMessages';
```

Comment:
I don't see where you are using this `userRoleMessage` you imported, where you using it on this module